src/constants.h

```diff
@@ -19,7 +19,7 @@
 //#define LAT_GRID_DIM_MAX 751 // Y
 //#define DEP_GRID_DIM_MAX 251 // Z
 #define SUB_MOD_DIM_MAX 50 // maximum number of entries in the 1d sub model arrays
-#define MAX_NUM_BASINS 40 // the maximum number of basins
+#define MAX_NUM_BASINS 50 // the maximum number of basins
```

Comment:
🤣

Reply:
I've noted one change needed so far. The smoothing lines need to be renamed so that they are just the basin_vXXpX.txt:

Napier1_v21p7.txt
HawkesBay1_v21p7.txt
GreaterWellington1_v21p7.txt
Porirua1_v21p7.txt

But when I did this I got a segmentation fault. Not sure how to fix. This is why we see a wall at the basin edge when it's mean to gradually change to the generic offshore basin.